.github/workflows/ci.yaml

```diff
@@ -20,3 +20,33 @@ jobs:
         run: |
           mkdir credentials && touch credentials/github-app-backstage-humanitec-credentials.yaml
           docker image build -t backstage:latest .
+
+  validate-score:
+    runs-on: ubuntu-latest
+    env:
+      HUMCTL_VERSION: '0.23.0'
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+      - uses: humanitec/setup-cli-action@v1
+        with:
+          version: ${{ env.HUMCTL_VERSION }}
+      - name: humctl score validate - backstage
+        run: |
+          humctl score validate --local --strict --org ignored \

```

Comment:
The `--org ignored` is required due to a bug in humctl, which will be fixed in the next release.